README.md

```diff
@@ -35,37 +40,26 @@ Use `feedrattler --help` to see all available options.
-│                                                        the current shell, to │
-│                                                        copy it or customize  │
-│                                                        the installation.     │
-│ --help                                                 Show this message and │
-│                                                        exit.                 │
-╰──────────────────────────────────────────────────────────────────────────────╯
+ Usage: feedrattler [OPTIONS] FEEDSTOCK_NAME [GITHUB_USERNAME]
+
+╭─ Arguments ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ *    feedstock_name       TEXT               [default: None] [required]                                                                 │
+│      github_username      [GITHUB_USERNAME]  [default: None]                                                                            │
+╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+╭─ Options ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ --use-pixi                       --no-use-pixi                                         [default: use-pixi]                              │
+│ --local-clone-dir                                                    TEXT              [default: None]                                  │
+│ --local-clone-dir-force-erase    --no-local-clone-dir-force-erase                      [default: no-local-clone-dir-force-erase]        │
+│ --branch-name                                                        TEXT              [default: convert_feedstock_to_v1_recipe_format] │
+│ --rerender                       --no-rerender                                         [default: rerender]                              │
+│ --enable-rerender-logs           --no-enable-rerender-logs                             [default: no-enable-rerender-logs]               │
+│ --log-level                                                          TEXT              [default: INFO]                                  │
+│ --github-token                                                       TEXT              [env var: GITHUB_TOKEN] [default: None]          │
+│ --dotenv                                                             TEXT              [default: None]                                  │
+│ --clone-type                                                         [auto|ssh|https]  [default: auto]                                  │
+│ --help                                                                                 Show this message and exit.                      │
+╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+
```

Comment:
Sorry to be picky, but you mind copy/pasting again with a reduced terminal width?

Reply:
Not at all! This definitely didn't work on the README. I think this looks reasonable now:

https://github.com/chrisburr/feedrattler/tree/use-ssh-push?tab=readme-ov-file#feedrattler-